pkg/odogenerator/ingressgenerator.go

```diff
@@ -26,12 +26,12 @@ func getNetworkingV1IngressSpec(ingressSpecParams generator.IngressSpecParams) *
 									Service: &v1.IngressServiceBackend{
 										Name: ingressSpecParams.ServiceName,
 										Port: v1.ServiceBackendPort{
-											Name:   fmt.Sprintf("%s%d", ingressSpecParams.ServiceName, ingressSpecParams.PortNumber.IntVal),
```

Comment:
Why are we removing this?

Reply:
Because having both fields in networking v1 ingress gives a schema related error

Reply:
Basically, you can have only one of those in the struct

Reply:
You can see similar aciton taken on library side https://github.com/devfile/library/blob/main/pkg/devfile/generator/utils.go#L358 based on my issue to get the generator in the library

Reply:
Thanks for the clarification :+1: